### ISSUE TITLE
Broken since 2.1

### ISSUE THREAD

**Issue by ornew** [facebookresearch/sam2#350]
Loading a model from HuggingFace fails in 2.1. It worked in 0f6515a.

```python
from sam2.sam2_image_predictor import SAM2ImagePredictor

sam2_predictor = SAM2ImagePredictor.from_pretrained("facebook/sam2-hiera-base-plus")
```

trace

```
Traceback (most recent call last):
  File "/Users/user/src/.venv/bin/x", line x, in <module>
    sys.exit(main())
  File "/Users/user/src/xxx/__init__.py", line x, in main
    sam2_predictor = SAM2ImagePredictor.from_pretrained(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/sam2/sam2_image_predictor.py", line 82, in from_pretrained
    sam_model = build_sam2_hf(model_id, **kwargs)
  File "/Users/user/src/.venv/lib/python3.10/site-packages/sam2/build_sam.py", line 126, in build_sam2_hf
    return build_sam2(config_file=config_name, ckpt_path=ckpt_path, **kwargs)
  File "/Users/user/src/.venv/lib/python3.10/site-packages/sam2/build_sam.py", line 69, in build_sam2
    cfg = compose(config_name=config_file, overrides=hydra_overrides_extra)
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/compose.py", line 38, in compose
    cfg = gh.hydra.compose_config(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/hydra.py", line 594, in compose_config
    cfg = self.config_loader.load_configuration(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/config_loader_impl.py", line 142, in load_configuration
    return self._load_configuration_impl(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/config_loader_impl.py", line 253, in _load_configuration_impl
    defaults_list = create_defaults_list(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 745, in create_defaults_list
    defaults, tree = _create_defaults_list(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 715, in _create_defaults_list
    defaults_tree = _create_defaults_tree(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 356, in _create_defaults_tree
    ret = _create_defaults_tree_impl(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 457, in _create_defaults_tree_impl
    return _expand_virtual_root(repo, root, overrides, skip_missing)
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 280, in _expand_virtual_root
    subtree = _create_defaults_tree_impl(
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 464, in _create_defaults_tree_impl
    config_not_found_error(repo=repo, tree=root)
  File "/Users/user/src/.venv/lib/python3.10/site-packages/hydra/_internal/defaults_list.py", line 799, in config_not_found_error
    raise MissingConfigException(
hydra.errors.MissingConfigException: Cannot find primary config 'configs/sam2/sam2_hiera_b+.yaml'. Check that it's in your config search path.
```

**Comment by b1ck0** [comment]
same here ... the configs/ subdirectory is not included in the package/wheel at all ... check setup.py

**Comment by haithamkhedr** [comment]
> same here ... the configs/ subdirectory is not included in the package/wheel at all ... check setup.py

Will submit a patch to include the configs as part of the package in case of non-editable installation. For now please install as editable following the instructions [here](https://github.com/facebookresearch/sam2?tab=readme-ov-file#installation)

**Comment by haithamkhedr** [comment]
This should be resolved in #353. Please reinstall to aadd all the configs to the package.
